getting-started/templates/systemlink-values.yaml

```diff
@@ -343,6 +343,77 @@ testmonitorservice:
       ##
       certificateSubPath: *postgresCertificateFileName
 
+## Configuration for the Alarm Service
+##
+alarmservice:
+  ## Per-replica rate limiting configuration
+  rateLimits:
+    ## Configuration for the global rate limiter that applies to all requests
+    ## @param rateLimits.global.enabled Enable the rate limiter
```

Comment:
I don't know the convention here, but do you think we should either remove the `@param rateLimits.*` part or prefix them with `alarmservice.` so they match where they are in this file?

Reply:
It's probably best to remove them. There are a couple instances of the `@param` syntax in this file, but they're in the minority. We're also inconsistent -- we use it for `global.ingress.ui.annotations`, but not for `global.ingress.api.annotations`, for example.

```
  ingress:
    ## Ingress settings that apply to the apiHosts endpoints.
    ##
    api:
      ## Annotations for the ingress.
      ##
      annotations: {}
    ## Ingress settings that apply to the hosts endpoints.
    ##
    ui:
      ## @param global.ingress.ui.annotations Annotations for the ingress.
      ##
      annotations: {}
 ```